integration/tests/karpenter/karpenter_test.go

```diff
@@ -61,14 +67,27 @@ var _ = Describe("(Integration) Karpenter", func() {
 				).
 				WithoutArg("--region", params.Region).
 				WithStdin(clusterutils.ReaderFromFile(clusterName, params.Region, "testdata/cluster-config.yaml"))
-			Expect(cmd).To(RunSuccessfully())
+			// For dumping information, we need the kubeconfig. We just log the error here to
```

Comment:
@cPu1 hmm, I don't really like this one. I need the config so even if this command fails I need it. It might fail too of course if the whole thing failed. But in case just the helm install failed I need this config. Maybe this could be a Matcher.

Reply:
I think it's fine for now as it's only for debugging purposes, since we'll remove it once we've figured out the issue.